data/crac-creation/crac-creator-csa-profiles/src/test/java/com/farao_community/farao/data/crac_creation/creator/csa_profile/crac_creator/CsaProfileCracCreatorTest.java

```diff
@@ -778,11 +777,11 @@ public void testTC1ImportPstRangeActions() {
         assertEquals("36b83adb-3d45-4693-8967-96627b5f9ec9", eliaRa1.getNetworkElement().getId());
         assertEquals(10, eliaRa1.getInitialTap());
         assertEquals(1, eliaRa1.getRanges().size());
-        assertEquals(5., eliaRa1.getRanges().get(0).getMinTap());
-        assertEquals(20., eliaRa1.getRanges().get(0).getMaxTap());
-        assertEquals(2, eliaRa1.getUsageRules().size());
-        assertEquals(CURATIVE, eliaRa1.getUsageRules().get(0).getInstant());
-        assertEquals("493480ba-93c3-426e-bee5-347d8dda3749", ((OnContingencyStateImpl) eliaRa1.getUsageRules().get(0)).getState().getContingency().get().getId());
+        assertEquals(5., eliaRa1.getRanges().iterator().next().getMinTap());
+        assertEquals(20., eliaRa1.getRanges().iterator().next().getMaxTap());
+        assertEquals(1, eliaRa1.getUsageRules().size());
+        assertEquals(CURATIVE, eliaRa1.getUsageRules().iterator().next().getInstant());
+       // TODO waiting for PO to check US, after implementation of CSA11 behaviour changed, assertEquals("493480ba-93c3-426e-bee5-347d8dda3749", ((OnContingencyStateImpl) eliaRa1.getUsageRules().iterator().next()).getState().getContingency().get().getId());
```

Comment:
OK for behavior change, but do check the type of the usage rule in the test

Reply:
In this case the test will not match the test case in the US of import of PstRangeActions (OnFlowConstraint prevailed)
As we intend to separate subjects in unit tests i m not sure if it's a good idea to test OnFlowConstraint here

Reply:
ok i get your point